float/src/main/java/com/novoda/floatschedule/convert/GithubUserConverter.java

```diff
@@ -0,0 +1,13 @@
+package com.novoda.floatschedule.convert;
+
+import java.util.List;
+
+public interface GithubUserConverter {
+    
+    List<String> getGithubUsers() throws Exception;
```

Comment:
`NoMatchFoundException` can be runtime and ignored in the signature? 

What about if the original catches its own IOException and throws NoMatchFoundException? bit cheeky but what do you think